### PR TITLE
seastore: add nvme commands to nvme device class

### PR DESCRIPTION
**Pull request by myoungwon**
Signed-off-by: Jinyong Ha <jy200.ha@samsung.com>
Signed-off-by: Myoungwon Oh <myoungwon.oh@samsung.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/crimson/os/seastore/random_block_manager/nvmedevice.h [hunk 1]
```diff
@@ -37,28 +65,121 @@ namespace crimson::os::seastore::nvme_device {
...
-    nvme_identify_command_t identify_cmd;
+  union {
+    nvme_passthru_cmd common;
+    nvme_identify_command_t identify;
   };
 
   static const uint8_t OPCODE_IDENTIFY = 0x06;
 };
 
-using NVMePassThroughCommand = nvme_passthru_cmd;
```
Comment (athanatos) [inline src/crimson/os/seastore/random_block_manager/nvmedevice.h:61]: A quick one line summary of what each of these structures is used for might help.

**Comment by athanatos** [comment]
Other than formatting nits, seems reasonable.  Ping me when you're ready to remove the WIP.

**Comment by myoungwon** [comment]
I think this is ready for review. We'll take into account how RBM makes full use of these feature in next PR.

**Comment by myoungwon** [comment]
@athanatos I think this is ready for review. We'll take into account how RBM makes full use of these feature in next PR.

**Review by athanatos — APPROVED** [review]

**Pull request merged**